### PR TITLE
Read twilio and nexmo credentials from files

### PR DESCRIPTION
**Pull request by tiago-loureiro**
Read secrets from files instead of directly from command line arguments

### CODE REVIEW

**Comment by tiago-loureiro** [comment]
Ah, well spotted: thanks! Fixed with: 8f819ab4c19c47968a4218e97a51d0c50000e42e

**Comment by tiago-loureiro** [comment]
Made some changes based on feedback, let me know if this reads better

File: services/brig/test/resources/nexmo.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+key: "dummy"
```
Comment (jschaul) [inline services/brig/test/resources/nexmo.txt]: is this file `yaml`? maybe change the extension from `.txt` to `.yaml`?

Comment (tiago-loureiro) [reply services/brig/test/resources/nexmo.txt]: 👍

**Pull request merged**